framework/include/cppmicroservices/Bundle.h

```diff
@@ -441,7 +442,8 @@ class US_Framework_EXPORT Bundle
    * @return A list of ServiceReference objects for all services this
    * bundle is using.
    *
-   * @throws std::logic_error If this bundle has been uninstalled.
+   * @throws std::logic_error If this bundle has been uninstalled, if
+   * the ServiceRegistrationBase object is invalid, or if the service is unregistered.
```

Comment:
There's an inconsistency with indentation of the newline after the ```@throw``` line. See other instances of ```@throw```